content/applications/hr/fleet/service.rst

```diff
@@ -86,61 +298,219 @@ clicked, sort the data in different ways.
 
 .. image:: service/views.png
    :align: center
-   :alt: The icons in the top right corner than cn be clicked to present the information in
+   :alt: The icons in the top right corner than can be clicked to present the information in
          different ways.
 
 List view
 ~~~~~~~~~
 
-The default view of the services is the list view. This presents all the services in chronological
-order, from oldest to newest, in a list.
+The default view of the service records is a list view. This presents all the services, first
+grouped alphabetically by type of service, then grouped by status.
+
+The information can be re-sorted by any column. At the top of each column, hover over the column
+name, and an arrow appears in the far-right of that column. Click the arrow to sort the data by that
+specific column.
+
+The default sorting is in descending alphabetical order (A to Z), represented by a :guilabel:`⌄
+(down arrow)`. Click the :guilabel:`⌄ (down arrow)` to reverse the alphabetical order (Z to A). The
+:guilabel:`⌄ (down arrow)` changes to an :guilabel:`^ (up arrow)`.
 
-The information can be sorted by any column. At the top of each column, hover over the column name.
-An arrow appears to the right of the name. Click the arrow to sort the data by that specific column.
+The two exceptions to this sorting are the default :guilabel:`Date` column and the :guilabel:`Cost`
+column. The :guilabel:`Date` column sorts the information in chronological order (January to
+December) instead of alphabetical order, while the :guilabel:`Cost` column sorts the information by
+repair price, from lowest to highest.
 
-The default sorting is descending alphabetical order (A to Z). Click the arrow again to reverse the
-order, and go in reverse alphabetical order (Z to A). The one exception to this sorting is the
-default :guilabel:`Date` column, which sorts the information in chronological order (January to
-December) instead of alphabetical order.
+Add a service
+*************
+
+To add a service record from the list view, click the :guilabel:`New` button, and a service form
+loads. :ref:`Enter all the information <service/service-form>` on the service form.
+
+The form automatically saves as data is entered.
 
 Kanban view
 ~~~~~~~~~~~
 
 To view services by their stage, click on the :guilabel:`Kanban icon`, which is the second icon in
```

Comment:
The only reason I thought I could use "Kanban" icon in a gui is when you rollover the icon, it says Kanban- which is why I thought I could use the name for the button. I am not in 100% alignment with your suggestion, only because when we say click "X", X has the gui because it's what you see on the UI - and that always get the gui.  Having the gui so far after the "click this" feels odd to me, since it is a real button they click. I honestly am waffling more over "icon" vs "button" so I changed that a bit around. But that is my logic for why I did it this way.  I do see what you mean, and the gui could even be applied to both- but I do prefer the actual button being named (since it has a name).